doc/faq.rst

```diff
@@ -26,6 +26,41 @@ See :ref:`contributing`. Before wanting to add a new algorithm, which is
 usually a major and lengthy undertaking, it is recommended to start with :ref:`known
 issues <easy_issues>`.
 
+What's the best way to get help on scikit-learn usage?
+--------------------------------------------------------------
+**For general machine learning questions**, please use
+`Cross Validated <http://stats.stackexchange.com>`_ with the ``[machine-learning]`` tag.
+
+**For scikit-learn usage questions**, please use `Stack Overflow <http://stackoverflow.com>`_
```

Comment:
Should we have the tagged page as the link? 

http://stackoverflow.com/questions/tagged/scikit-learn